Makefile

```diff
@@ -60,9 +60,10 @@ else
     $(error Unknown CPU architecture ${ARCH} detected)
 endif
 
+MODULUS_GIT_HASH = $(shell git rev-parse --short HEAD)
```

Comment:
Do we want the `--short` here? Why not keep the full hash?

Reply:
I think it was more from presentation standpoint. The full hash is 40 characters long, which I feel might be a bit too much. The short hash should still be resolvable with fallbacks present: https://stackoverflow.com/a/56012434

Alternative would be to make the ENV variable full hash and then truncate it only when printing in the header. I guess that can be done easily. What do you think is preferable?

Reply:
Actually short will also automatically try to ensure that uniqueness is maintained by automatically increasing the length if required. Resolving this.